src/tcp.c

```diff
@@ -3217,8 +3217,9 @@ relpTcpRcv(relpTcp_t *const pThis, relpOctet_t *const pRcvBuf, ssize_t *const pL
 			} else if(errno == ECONNRESET) {
 				pThis->pEngine->dbgprint((char*)"relpTcpRcv: read failed with errno ECONNRESET!\n");
 #if defined(_AIX)
-			} else if(errno == 0) {
-				// Set mode to Retry if errno is 0, this actually happens on AIX 10.x
+			} else if(	errno == 0 ||
+						errno == EINTR) {
```

Comment:
errno == EINPROGRESS is missing. I think I did see that from relpTcpRcv even though man page differs on codes of recv

Reply:
Did you receive that code during testing? According to DOC, this code only being used in connect() and not in recv() / send().

Reply:
I think assignment of 
errno = EINPROGRESS;
caused this as a side effect.

Reply:
> I think assignment of errno = EINPROGRESS; caused this as a side effect.

perhaps yes, but that assignment was removed so I think we should be good now with this PR.

Reply:
i agree. i didn't observe EINPROGRESS in recv / send after removal